Makefile

```diff
@@ -240,7 +240,7 @@ build-farm-consistency:
 	@echo "diffing ns-ttl-controller assets ..."
 	diff -Naup ./core-services/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml ./clusters/build-clusters/01_cluster/openshift/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml
 	@echo "diffing rpms-ocp assets ..."
-	for file in ./core-services/release-controller/rpms-ocp-*.yaml; do diff -Naup "$${file}" "./clusters/build-clusters/01_cluster/openshift/release-controller/$${file##*/}"; done
+	for file in ./core-services/release-controller/rpms-ocp-*.yaml; do diff -Naup "$${file}" "./clusters/build-clusters/common/ocp/$${file##*/}"; done
```

Comment:
this is pretty useess now, we basically diff that the symlink against the symlink target?

Reply:
Removed.